qemux86-64.robot

```diff
@@ -49,6 +49,14 @@ Running image
   Set Suite Variable    ${device_qemu_handle}    ${handle}
 Checking if device comes online in 60s (Trying every 10s)
   Wait Until Keyword Succeeds    6x    10s    Device ${device_uuid} is online
+Check that backup files are not found in the image 
+  ${LOOPDEVICE} =   Set up loop device for "${image}"
+  Set Suite Variable    ${path_to_loop}    /dev2/loop${LOOPDEVICE}
```

Comment:
Maybe we can change this to Test Variable? so that it is available only in test - same with line: 56